cunumeric/module.py

```diff
@@ -4706,7 +4719,7 @@ def cumsum(
 
     dtype : dtype, optional
         Type of the returned array and of the accumulator in which the elements
-        are summed. If dtype is not specified, it defaults to the dtype of a,
+        are returned. If dtype is not specified, it defaults to the dtype of a,
```

Comment:
this edit doesn't seem to make much sense.

Reply:
a3a8593fed3c1f0534e2d2dc251d75967b9790c3